### PR TITLE
Use tagged version of busybox

### PR DESCRIPTION
**Pull request by kierenevans**
Avoid using `:latest` as it's unpredictable.

Less important due to only using `chown` from busybox.

### CODE REVIEW

File: src/_base/helm/app/templates/application/cron/deployment.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
       {{- if not (eq "" (include "application.volumes.wwwDataPaths" $)) }}
       initContainers:
       - name: cron-volume-permissions
-        image: busybox
+        image: busybox:1.36
```
Comment (andytson-inviqa) [inline src/_base/helm/app/templates/application/cron/deployment.yaml]: should be configurable if not latest

Comment (andytson-inviqa) [reply src/_base/helm/app/templates/application/cron/deployment.yaml]: 1.35/1.36 are marked "unstable" as well

Comment (kierenevans) [reply src/_base/helm/app/templates/application/cron/deployment.yaml]: Will sort :+1:

**Comment by kierenevans** [comment]
Build failed, will take a look

**Review by andytson-inviqa — APPROVED** [review]

File: src/_base/helm/app/templates/application/cron/deployment.yaml (hunk 1 above)
Comment (andytson-inviqa) [reply src/_base/helm/app/templates/application/cron/deployment.yaml]: erm, sorry as mentioned 1.35 also "unstable". 1.34 seems the most recent "stable"

Comment (kierenevans) [reply src/_base/helm/app/templates/application/cron/deployment.yaml]: 1.35 is the earliest supported tag on https://hub.docker.com/_/busybox

**Pull request merged**